src/main/resources/hudson/plugins/sonar/SonarGlobalConfiguration/config.jelly

```diff
@@ -11,8 +11,8 @@
       <div class="sonar-section">
       <f:repeatable var="inst" items="${instance.installations}" add="${%AddSonar}">
         <s:blockWrapper class="sonar-installation">
-          <f:entry title="${%Name}">
-            <f:textbox name="sonar.name" value="${inst.getName()}" checkUrl="'${rootURL}/descriptor/SonarGlobalConfiguration/checkMandatory?value='+escape(this.value)"/>
+          <f:entry title="${%Name}" field="name">
+            <f:textbox name="sonar.name" value="${inst.getName()}"/>
```

Comment:
https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/util/FormValidation.java#L625-L727 will do all the magic. For that reason I also had to rename the validation method.